workflows/seqqc.nf

```diff
@@ -38,6 +38,10 @@ ch_multiqc_custom_methods_description = params.multiqc_methods_description ? fil
 //
 include { INPUT_CHECK } from '../subworkflows/local/input_check'
 
+//
+// MODULE: Local modules
+//
+include { DOWNLOAD_SOURMASH_GATHER_DBS } from '../modules/local/download_sourmash_gather_dbs'
```

Comment:
Is this only written currently to download the databases for each new run? Or is there a way later we could add support that if we already have the databases downloaded we could "point" to them with an additional argument?

Reply:
only written to download db if it's not available for the current set of runs. It's a very small database, so I think this behavior is ok. we could change it to point but I don't know how to do that yet.